java/src/IceGridGUI/src/main/java/com/zeroc/IceGridGUI/Application/Adapter.java

```diff
@@ -3,9 +3,7 @@
 package com.zeroc.IceGridGUI.Application;
 
 import com.zeroc.IceGrid.AdapterDescriptor;
```

Comment:
The rest of this PR is removing the unused imports that checkstyle caught.
Good job to checkstyle.